### ISSUE TITLE
Uninitialized constant Ability in CategoriesController

### ISSUE THREAD

**Issue by klenga** [rubysherpas/forem#186]
Hi-

I just installed forem today and am able to do everything but view categories.  I get this error when I click on the link for the 'General' category.

```
NameError in Forem::CategoriesController#show

uninitialized constant Ability
Rails.root: /Users/klenga/Projects/surfbreakrentals

Application Trace | Framework Trace | Full Trace
Request

Parameters:

{"id"=>"1"}


Response

Headers:

None
```

I'm not sure why this would happen for just the categories controller.

**Comment by radar** [comment]
What happens if you fork forem and get that controller to point to `Forem::ApplicationController` rather than just `ApplicationController`? Both should work in identical ways, it's just that the way it is now should try loading the `Forem::ApplicationController` constant before it would try loading the actual `ApplicationController` constant.

If it _doesn't_ break when you reference `Forem::ApplicationController` then that means it's probably referencing the actual application's `ApplicationController` and that's a Bad Thing(tm).

**Comment by klenga** [comment]
thanks for the quick reply!  I tried what you suggested and it works now... I'm kind of a rails novice so I want to make sure I'm doing this correctly:

-forked forem to klenga/forem
-updated the CategoriesController
-changed my gemfile to gem 'forem', :git => "git://github.com/klenga/forem.git"
-ran bundle install

...the categories page now displays correctly.  If I switch back to radar/forem in the gemfile and bundle install, the error reappears.  Actually I get this error: 

> Routing Error
> 
> undefined local variable or method `load_and_authorize_resource' for Forem::CategoriesController:Class

initially, then it changes to the 'uninitialized constant' error if I navigate to a forum page then back to the category page.

**Comment by radar** [comment]
Alright, fairly easy fix then. One moment.

**Comment by radar** [comment]
Looks like it was just that controller, and so I've fixed it up in faeacc8b8ea66ff1bab03b6f98d35a7f1a545797. Thanks for taking the time to confirm that for me. You totally nailed it :)

**Comment by klenga** [comment]
awesome, thanks!
